index.d.ts

```diff
@@ -4,167 +4,75 @@
  */
 
 /**
- * Possible values of a shell. `false` and `undefined` mean no shell. `true`
- * means the default system shell, and any non-empty string configures a
- * particular shell.
- */
-type ShellOption = boolean | string | undefined;
-
-/**
- * Options for {@link escape} and {@link escapeAll}.
+ * Options for {@link Shescape}.
+ *
+ * @since 2.0.0
  */
-interface EscapeOptions {
+interface ShescapeOptions {
   /**
    * Whether or not to protect against flag and option (such as `--verbose`)
    * injection
    *
-   * @default false
-   * @since 1.7.0
+   * @default true
+   * @since 2.0.0
    */
   readonly flagProtection?: boolean;
 
   /**
    * Is interpolation enabled.
    *
-   * @default false
-   * @since 1.4.0
+   * @default true
+   * @since 2.0.0
    */
   readonly interpolation?: boolean;
 
   /**
-   * The shell to escape for.
+   * The shell to escape for.  `false` and `undefined` mean no shell. `true`
+   * means the default system shell, and any non-empty string configures a
+   * particular shell.
```

Comment:
There's 1 space too much here.

```suggestion
   * The shell to escape for. `false` and `undefined` mean no shell. `true`
   * means the default system shell, and any non-empty string configures a
   * particular shell.
```

Reply:
Resolved in 2b990d66d309fd17ab6520fd9e48dd5abf5f9a2f